info/structure.md

```diff
@@ -1 +1,47 @@
-# Program structure
\ No newline at end of file
+# Program structure
+
+The Climatematch Impact Scholars program runs between **October 2023** and **January 2024**.
+
+## **What is involved**
+- **Computing resources**. Thanks to our partnership with [2i2c](https://2i2c.org/), selected groups will be provided with continued access to the Climatematch JupyterHub until the end of the program with a possibility of further extension if the scholars and their mentor are interested in working towards a peer-reviewed publication. 
+
+- **Citable micropublication**. At the end of the program, the scholars will write up their results in a micropublication that will be publicized on the program website and assigned a DOI. The scholars will also be offered a chance to share their work in their native language to increase its impact in their community.
+
+- **Seminar presentation**. Also at the end of the program, the scholars will give a 20-minute virtual seminar that will be advertised on Climatematch social media and recorded for subsequent sharing on the course website.
+
+- **Mentorship**. We endeavour to match every team to a suitable mentor who connects with the scholars synchronously or asynchronously at least once a month and provides feedback on their research outputs. 
+
+- **Skill development workshops**. To assist with preparation of the micropublication and the seminar, we will be running workshops on [science writing](../workshops/science_writing.md) and [oral presentation](../workshops/oral_presentations.md).
+
+- **Community support**. There will be regular check-ins from the program organisers on the #cisp-community forum channel on Discord for participants to share both successes and struggles.
+
+We are actively working to provide participants with further professional and academic opportunities and additionally encourage them to take part in Climatematch seminars, journal club meetings, and professional development survey series. These are not strictly part of the program, but are aligned with the scholars' project topics and career aspirations.
+
+---
+## **Program timeline**
+<table style="width:60%">
+<tr>
+    <td><b>October 2023</b></td>
+    <td>Welcome Event for scholars and mentors</td>
+</tr>
+<tr>
+    <td><b>December 2023</b></td>
+    <td>Workshop on science writing</td>
+</tr>
+<tr>
+    <td><b>Early January 2024</b></td>
+    <td>Workshop on oral presentation</td>
+</tr>
+<tr>
+    <td><b>12th January 2024</b></td>
+    <td>Research output submission deadline</td>
+</tr>
+<tr>
+    <td><b>Late January 2024</b></td>
+    <td>Feedback on research outputs</td>
+</tr>
+<tr>
+    <td><b>February 2024</b></td>
+    <td>Micropublication release and seminar presentations</td>
+</tr>
```

Comment:
Tried to address this in PR #2. @zanemit please take a look and let me know what you think. Left the word tentative there so that it leaves room for changes.

Reply:
I like the word "tentative" :) 
I am fully on board with your suggested changes. I think it would be great if the organising team was able to reach a consensus on the points I listed in https://github.com/ClimateMatchAcademy/impact-scholars/pull/2#pullrequestreview-1654801502 before the Welcome Events and we could announce the website with those decisions reflected in the timeline, but let's see how the discussion unfolds.